holmes.go

```diff
@@ -73,11 +73,13 @@ func main() {
 	//only prepend the protocol when the user provided one
 	if *protocol != "" {
 		baseURL = *protocol + "://"
+	} else {
+		baseURL = "//"
 	}
 
 	//only prepend the host when the user provided one
 	if *host != "" {
-		baseURL = *host
+		baseURL = baseURL + *host
```

Comment:
We need to fix this in a patch release. Can you pls apply this one in a separate PR so that we can release 1.10.0 immediately? #14